stable/graylog/Chart.yaml

```diff
@@ -1,6 +1,7 @@
+apiVersion: v1
 name: graylog
 home: https://www.graylog.org
-version: 0.1.4
+version: 1.0.0
```

Comment:
Shouldn't this be `0.1.5`?

Reply:
we agreed some time ago that all charts in the stable folder should be at least 1.0.0 @mattfarina